components/ui/themes/DefaultTheme.ts

```diff
@@ -474,6 +477,17 @@ export const DefaultTheme = {
       color: Colors.Red,
       fontSize: 18,
     },
+    QrView: {
+      padding: 6,
+      backgroundColor: 'white',
+      marginTop: 20,
+      borderRadius: 10,
+      elevation: 10,
```

Comment:
I think this elevation wouldn't work. If once checked from line-36 in the themes, we are given max. level for elevation is 6.
Better once check this in both themes.

Reply:
changed elevation to 6